core/math/math.sgl

```diff
@@ -19,4 +19,18 @@ var pi = 3.1415926535897932;
 var sqrt1_2 = 0.7071067811865476;
 
 // Square root of 2
-var sqrt2 = 1.4142135623730951;
\ No newline at end of file
+var sqrt2 = 1.4142135623730951;
+
+function min(a, b) {
+    if (b < a) return b;
```

Comment:
Please separate the bodies onto a newline

Reply:
Sure.